src/main/java/at/hannibal2/skyhanni/config/features/garden/SensReducerConfig.java

```diff
@@ -0,0 +1,32 @@
+package at.hannibal2.skyhanni.config.features.garden;
+
+import at.hannibal2.skyhanni.config.core.config.Position;
+import com.google.gson.annotations.Expose;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.moulberry.moulconfig.annotations.ConfigEditorSlider;
+import io.github.moulberry.moulconfig.annotations.ConfigOption;
+import io.github.moulberry.moulconfig.observer.Property;
+
+public class SensReducerConfig {
+    @Expose
+    @ConfigOption(
+        name = "Enable",
+        desc = "Lowers mouse sensitivity while in the garden, and a farming tool is held.")
+    @ConfigEditorBoolean
+    public boolean enabled = false;
```

Comment:
This is not the main toggle for the whole category. the name enabled feels wrong. maybe use auto enable or something?